### PR TITLE
Cleaner code

### PR DESCRIPTION
**Pull request by MattBDev**
- Removed two unused variables in PlotMe.java
- Merged 'else if' statements in PMCommand.java
- Removed unnecessary code from PMCommand.java
- Updated dependencies in pom.xml
- Added text to README.md
- Removed .idea files because not everyone uses IntelliJ. These files should not be in the GitHub Repository. 

Additions will be added below as committed.

### CODE REVIEW

**Comment by MattBDev** [comment]
I feel like these changes benefit PlotMe and improve upon the plugin. I encourage you to take a look at my changes and hopefully pull them.

**Comment by MattBDev** [comment]
New Additions:
- Removed PREFIX from logger messages. Logger prefix is now provided by Bukkit automatically due to commit 5963cb44295273f3c340a08f0fd11736db601d1c

File: pom.xml [hunk 1]
```diff
@@ -91,17 +96,21 @@
...
             <artifactId>worldedit</artifactId>
-            <version>5.5.8-SNAPSHOT</version>
+	        <version>5.5.8</version>
+	        <type>jar</type>
         </dependency>
         <dependency>
             <groupId>net.milkbowl.vault</groupId>
-            <artifactId>Vault</artifactId>
-            <version>1.2.32</version>
+	        <artifactId>VaultAPI</artifactId>
```
Comment (MattBDev) [inline pom.xml:111]: VaultAPI is the prefered method of implementing vault.

File: pom.xml [hunk 2]
```diff
@@ -91,17 +96,21 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
-            <version>1.7.9-R0.1-SNAPSHOT</version>
+	        <version>1.7.10-R0.1-SNAPSHOT</version>
+	        <type>jar</type>
+	        <scope>provided</scope>
```
Comment (MattBDev) [inline pom.xml:101]: Bukkit is provided

File: pom.xml [hunk 3]
```diff
@@ -91,17 +96,21 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
-            <version>1.7.9-R0.1-SNAPSHOT</version>
+	        <version>1.7.10-R0.1-SNAPSHOT</version>
```
Comment (MattBDev) [inline pom.xml:99]: Update to 1.7.10

**Comment by MattBDev** [comment]
May require Java 7 but servers should be running Java 7 anyways due to performance and security enhancements.
(According to MCStats.org less than 1% of servers running PlotMe use Java 6)
Please comment if that is an issue because this can be fixed.

**Comment by ZachBora** [comment]
I'll check this when I get back from vacation Friday.

**Comment by MattBDev** [comment]
Great. :+1:

**Pull request merged**